src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -159,25 +146,30 @@ class CompanyStep extends React.Component {
             errors.companyPhone = true;
         }
 
-        _.each(this.requiredFields, (inputKey) => {
-            if (ValidationUtils.isRequiredFulfilled(this.state[inputKey])) {
-                return;
-            }
+        ReimbursementAccount.setBankAccountFormValidationErrors(errors);
 
-            errors[inputKey] = true;
-        });
-        BankAccounts.setBankAccountFormValidationErrors(errors);
         return _.size(errors) === 0;
     }
 
     submit() {
         if (!this.validate()) {
-            BankAccounts.showBankAccountErrorModal();
             return;
         }
 
         const incorporationDate = moment(this.state.incorporationDate).format(CONST.DATE.MOMENT_FORMAT_STRING);
         BankAccounts.setupWithdrawalAccount({
+            bankAccountID: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'bankAccountID', 0),
+
+            // Fields from bankAccount step
+            routingNumber: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'routingNumber'),
+            accountNumber: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'accountNumber'),
+            bankName: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'bankName'),
+            plaidAccountID: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'plaidAccountID'),
+            isSavings: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'isSavings'),
+            plaidAccessToken: ReimbursementAccountUtils.getDefaultStateForField(this.props, 'plaidAccessToken'),
+            acceptTerms: true,
```

Comment:
NAB I don't think we should hardcode this value

Reply:
I think we can remove this now since it's being set on the API side